### PR TITLE
[WIP] Documentation fixes

### PR DESCRIPTION
**Pull request by okdas**
After a few PRs were merged, I went through the readme as I would be deploying from scratch. Some changes were necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,30 +111,15 @@ cp .env.sample .env
 Update `NODE_HOSTNAME` in `.env` to the IP address or hostname of your node. For example:
 
 ```bash
-sed -i -e  s/NODE_HOSTNAME/69.42.690.420/g .env
+sed -i -e s/NODE_HOSTNAME=/NODE_HOSTNAME=69.42.690.420/g .env
 ```
 
 ## A. Deploying a Full Node
 
-### 1. Download Network Genesis
```
Comment (okdas) [inline README.md:119]: No longer needed with the script provided by @breezytm

File: README.md [hunk 2]
```diff
@@ -111,30 +111,15 @@ cp .env.sample .env
...
-into the `poktrolld/config` directory:
-
-E.g. Testnet-validated:
-
-```bash
-curl https://raw.githubusercontent.com/pokt-network/pocket-network-genesis/master/poktrolld/testnet-validated.json > poktrolld-data/config/genesis.json
-```
-
-### 2. Launch the Node
+### 1. Launch the Node
```
Comment (Olshansk) [inline README.md:118]: Can you make sure the ToC is updated? 

It's just a save operation for me: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one

File: appgate-server-example/config/appgate_config.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # This can be used by the Cosmos client SDK, event subscriptions, etc...
 query_node_rpc_url: tcp://poktrolld:26657
 # Pocket node URL that exposes the Cosmos gRPC service, dedicated to querying purposes.
-query_node_grpc_url: tcp://poktrolld:26658
+query_node_grpc_url: tcp://poktrolld:9090
```
Comment (Olshansk) [inline appgate-server-example/config/appgate_config.yaml:6]: Why are we updateing `grpc` for not `normal rpc`

File: hetzner_cheasheet.md [hunk 4]
```diff
@@ -76,6 +76,6 @@ sed -i -e s/YOUR_NODE_IP_OR_HOST/.../g .env
 source helpers.sh
 
 poktrolld keys add relayminer-1
-export RELAYMINER_ADDR=pokt15l3gamxkz5c5pz9ynk0xqmkq6m07k5jcn89cl9
+export SUPPLIER_ADDR=pokt15l3gamxkz5c5pz9ynk0xqmkq6m07k5jcn89cl9
```
Comment (Olshansk) [inline hetzner_cheasheet.md:79]: 👍

**Review by Olshansk — CHANGES_REQUESTED** [review]

File: appgate-server-example/config/appgate_config.yaml (hunk 3 above)
Comment (okdas) [reply appgate-server-example/config/appgate_config.yaml:6]: @Olshansk default rpc port is `26657` so no need to change it. Default grpc port is `9090`.

**Pull request merged**